libs/ledger-live-common/src/generated/types.ts

```diff
@@ -57,7 +57,7 @@ import type {
   TransactionRaw as filecoinTransactionRaw,
   TransactionStatus as filecoinTransactionStatus,
   TransactionStatusRaw as filecoinTransactionStatusRaw,
-} from "../families/filecoin/types";
+} from "@ledgerhq/coin-filecoin/types/index";
```

Comment:
Do we need to target `/index`? Might be resolved automatically by node. Otherwise, it means we have an issue with our declarations in the `package.json`

Reply:
for this one if I do not have `/index` then the build is failing